### ISSUE TITLE
Accessibility improvements in Buttons

### ISSUE THREAD

**Issue by yceballost** [Telefonica/mistica-design#1443]
- [x] Update specs
- [x] [Android ticket](https://jira.tid.es/browse/ANDROID-13801)
- [x] [iOS ticket](https://jira.tid.es/browse/IOS-9377)
- [x] [Web ticket ](https://jira.tid.es/browse/WEB-1561)

### Discussed in https://github.com/Telefonica/mistica-design/discussions/1435

<div type='discussions-op-text'>

<sup>Originally posted by **Ro-Matos** August 30, 2023</sup>
### Goal

- 1 - Add a optional prop `aria-label` to Button component. Sometimes the semantic label is tied to the context that there isn't enough info for screen readers, should be possible to allow including an aria-label that provides additional context.

Example: Visible label is "See more" and aria-label is "See more products".

- 2 - Add prop `aria-disabled` to buttons in 'disabled' state. The aria-disabled state indicates that the element is perceivable but disabled. Reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled

Some screen-readers jump the button when only the prop `disabled` is use.

### Context

In our accessibility specs of flows, developed by the design teams of APP Vivo, this cases have frequently documented. But the developers are not able to assign the properties in the code.

### Impact

Improvements and compliance with accessibility guidelines.

### Discussion Questions

None

### Do you have a visual proposal?

_No response_</div>
